Assets/code_src/PlayerCameraController.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCameraController : MonoBehaviour
+{
+    public float lerpSpeed;
+
+    private Transform player;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void LateUpdate()
+    {
+        if (!player)
+        {
+            player = FindObjectOfType<PlayerController>().gameObject.transform;
+        }
+
+        if (!player)
+        {
+            return;
+        }
+
+        Vector2 currentPosition = new Vector2(transform.position.x, transform.position.y);
```

Comment:
Yea, I can move it to a separate function but I think we should call it `UpdateCameraPosition()` since we are moving the camera around not the player.